confAllChambers.py

```diff
@@ -52,6 +58,8 @@ def launchArgs(shelf,link,slot,run,vt1,vt1bump,config,cName,ztrim):
 
     parser.add_option("--config", action="store_true", dest="config",
                       help="Set Configuration from simple txt files", metavar="config")
+    parser.add_option("--dictConfig", action="store_true", dest="dictConfig", default=False,
+                      help="Configure VFATs to custom chamber_default values", metavar="dictConfig")
```

Comment:
Yes the case structure you showed above is probably a better way to apply it; the goal here was just to provide an example.  It's more likely that `confChamber.py` is the correct place to put this case structure since this is performing the actual writing of registers; the job of `confAllChambers.py` is to ensure the call of `confChamber.py` for a given link is correct.

In any case, you get the idea.